ui/src/shared/components/notifications/Notifications.tsx

```diff
@@ -1,53 +1,99 @@
 import React, {PureComponent} from 'react'
+import {Link} from 'react-router'
 import {connect} from 'react-redux'
-import {Notification as NotificationType} from 'src/types/notifications'
-import Notification from 'src/shared/components/notifications/Notification'
+import {get} from 'lodash'
 
-interface Props {
+//Actions
```

Comment:
*cough* this seems like a great thing to split up into a separate PR that can get merged ahead of this one.

Reply:
totally agree and thank you so much. I definitely tried cherry picking apart, but decided to just go for it in this case 😬